### DISCUSSION TITLE
Error from running a file from Github codespaces on Vscode

### DISCUSSION THREAD

**Discussion by WilliamKusuma** [community/community#133600] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Good Morning!
Hi! I am sorry about this boring question I'm new in this field and sorry for my broken English. 
I got a question about how to run or start debugging for my html file? (its like challenges i got about html from user [lordelogos](https://github.com/lordelogos) repositories). 
I'm doing the challenge on my repositories from vscode connected via "GitHub Codespaces" . Normally to preview or run html file from my local drive I'm just using F5 or "Show Preview" or "Live Server". But when I'm connect vscode with github codespaces (to completed the challenge from my repositories) somehow the "show preview" or "live server" extension are missing. So I'm trying F5 to start debugging the challenge but its show a error on vscode (Could not read source map for chrome-error://chromewebdata/: Unexpected 503 response from chrome-error://chromewebdata/neterror.rollup.js.map: Unsupported protocol "chrome-error:") I attached some screenshots.
I'm have done some search before but still error comes out (sorry probably I'm missed some discussion that have solution for it) or did its need a extension?
I hope to get a solution (or at least key word to search on google) to this problem from you guys who may have same experience or understand about this .
I'm sorry if my words are confusing  and Thank you for any help.

![Screenshot 2024-07-26 102804](https://github.com/user-attachments/assets/5a90e935-87d3-4e30-83aa-7a873fb0401e)
![Screenshot 2024-07-26 102909](https://github.com/user-attachments/assets/aef04c87-80ec-4fa0-a081-f41cc94d2f6e)

**Comment by SohanKafle** [comment, marked as answer]
Hi [WilliamKusuma](https://github.com/WilliamKusuma),

To preview your HTML file in GitHub Codespaces, you can install the "Live Server" extension manually since it's not included by default.

Do these following steps:
<b>1. Install Live Server Extension:</b>
 Open Extensions view (Ctrl+Shift+X).
 Search for "Live Server" and install it.

<b>2. Start Live Server:</b>
Right-click your HTML file and select "Open with Live Server".

If you encounter any issues, ensure your Codespace has the necessary permissions and resources.

**Reply by WilliamKusuma** [reply]
Hi [SohanKafle](https://github.com/SohanKafle),
Thank you very much for the solution you provided so that now I can run the preview. I thought the live server extension couldn't be used for github codespaces (and it was my fault also I didn't check whether it can be installed on it or not), it turns out it's just not available in the default installation so i have to install it again. Btw Thank you very much for the solution!
